bin/weewx/drivers/vantage.py

```diff
@@ -1481,6 +1502,14 @@ def _archive_datetime(datestamp, timestamp):
                       0, 0, -1)                     # have OS guess DST
         # Convert to epoch time:
         ts = int(time.mktime(time_tuple))
+        # fix for hour mismatches at begin of summertime
+        # calculate a time tuple from the UTC time stamp
+        utc_time_tuple = time.localtime(ts)
```

Comment:
That's is where the problem lies, Tom. If the Vantage console would report its timestamps in UTC we wouldn't had a problem. What we do is convert vantaghe time stamp to UTC and then back (to the Vantage's format). During the first hour of the summertime these times won't be the same but differ 1 hour. No solution ever to fix this either, we have to accept this. Because of this one hour difference the stored record in the weewx database (in UTC) never can be translated back to the time stamp the vantage has of this record (one our differnce , yoy know).